Shared/Samples/Show coordinates in multiple formats/ShowCoordinatesInMultipleFormatsView.swift

```diff
@@ -0,0 +1,167 @@
+                        spatialReference: model.mapPoint.spatialReference
+                    )!)
+            }
+            CoordinateTextField(
+                title: "UTM",
+                text: $model.utmString
+            )
+            .onSubmit {
+                model.updateMapPoint(
+                    point: CoordinateFormatter.point(
+                        fromUTMString: model.utmString,
+                        spatialReference: model.mapPoint.spatialReference,
+                        conversionMode: .latitudeBandIndicators
+                    )!)
+            }
+            CoordinateTextField(
+                title: "USNG",
+                text: $model.usngString
+            )
+            .onSubmit {
+                model.updateMapPoint(
+                    point: CoordinateFormatter.point(
+                        fromUSNGString: model.usngString,
+                        spatialReference: model.mapPoint.spatialReference
+                    )!)
+            }
+            
+            // Create a map view to display the map.
+            MapView(map: model.map, graphicsOverlays: [model.graphicsOverlay])
+                .onSingleTapGesture { _, mapPoint in
+                    model.updateMapPoint(point: mapPoint)
+                }
+        }
+    }
+}
+
+struct CoordinateTextField: View {
+    /// The title string.
+    var title: String
+    
+    /// The text string.
+    @Binding var text: String
+    
+    var body: some View {
+        VStack(alignment: .leading) {
+            Text(title)
+                .padding([.leading, .top], 8)
+                .padding(.bottom, -5)
+            TextField("", text: $text)
+                .textInputAutocapitalization(.never)
+                .disableAutocorrection(true)
+                .textFieldStyle(.roundedBorder)
+                .padding([.leading, .trailing, .bottom], 8)
+        }
+    }
+}
+
+private extension ShowCoordinatesInMultipleFormatsView {
+    // The view model for the sample.
+    class Model: ObservableObject {
+        /// A map with an imagery basemap.
+        var map: Map
+        
+        /// The graphics overlay for the point graphic.
+        let graphicsOverlay: GraphicsOverlay
+        
+        /// A yellow cross graphic for the map point.
+        private let pointGraphic: Graphic
+        
+        /// The point on the map.
+        @Published var mapPoint: Point!
+        
+        /// The decimal degrees text.
+        @Published var latLongDDString = ""
+        
+        /// The degree minute seconds text.
+        @Published var latLongDMSString = ""
+        
+        /// The UTM text.
+        @Published var utmString = ""
+        
+        /// the USNG text.
+        @Published var usngString = ""
+        
+        init() {
+            map = Map(basemapStyle: .arcGISImageryStandard)
+            pointGraphic = Graphic(symbol: SimpleMarkerSymbol(style: .cross, color: .yellow, size: 20))
```

Comment:
These declarations can both be used where the property is declared they don't need to be in the init.